system-test/src/util/env.js

```diff
@@ -0,0 +1,41 @@
+const DOCKER_COMPOSE_FILE = '../docker-compose.yml -f ../docker-compose.st.yml'
```

Comment:
The `DockerCompose` function does support multiple compose files by passing an array of files.

```suggestion
const DOCKER_COMPOSE_FILE = ['../docker-compose.yml', '../docker-compose.st.yml']
```

Reply:
Thx!